### PR TITLE
Parallel full build tests

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: _targets.R
Comment (mem48) [inline _targets.R:1]: @Robinlovelace is this is duplication? The `get_routes` function already checks and reads saved routes.

**Review by mem48 — COMMENTED** [review]
is this is duplication? The get_routes function already checks and reads saved routes.

**Comment by Robinlovelace** [comment]
> is this is duplication? The get_routes function already checks and reads saved routes.

@mem48 As you would see if you tried the full build, it currently fails with 400k+ routes. I will look for a fix but currently the code in this branch fails to generate a full build. 

Can you reproduce the issue on your computer? I wonder if internet speed is an issue...

**Comment by Robinlovelace** [comment]
And to be clear, I mean trying on the #262 branch. See https://github.com/nptscot/npt/pull/262#issuecomment-1710023200 for the error I'm hitting which forced the changes in this branch (still not working).

**Comment by mem48** [comment]
I'll try a full build as my test are all running fine

**Comment by Robinlovelace** [comment]
> I'll try a full build as my test are all running fine

Great stuff. BTW I think I've fixed the issues with the timeout upstream.

**Comment by Robinlovelace** [comment]
https://github.com/cyclestreets/cyclestreets-r/commit/cc94cfbe15f15ac00635cabf4222b3845ea91ede

**Pull request closed without merging**